Python/omevv/v1/scripts/vSphereHostFirmwareUpdate.py

```diff
@@ -0,0 +1,284 @@
+        response: Response[Union[ErrorObject, int]] = \
+            trigger_update.sync_detailed(uuid=self.uuid, client=self.client, gid=omevv_group_id,
+                                         json_body=json_body)
+        response_content = json.loads(response.content.decode("utf-8").replace("'", '"'))
+        return response.parsed, response_content, response.status_code
+
+    def firmware_update_job_by_id(self, job_id):
+        response: Response[Union[ErrorObject, Job]] = \
+            get_update_service_job_by_id.sync_detailed(uuid=self.uuid, client=self.client, id=job_id)
+        response_dict = json.loads(response.content.decode("utf-8").replace("'", '"'))
+        return response_dict, response.status_code
+
+
+if __name__ == "__main__":
+    PARSER = argparse.ArgumentParser(description=__doc__,
+                                     formatter_class=argparse.RawTextHelpFormatter)
+    PARSER.add_argument("--ip", "-i", required=True, help="OME Appliance IP")
+    PARSER.add_argument("--vcusername", "-u", required=True,
+                        help="username of vcenter")
+    PARSER.add_argument("--vcpassword", "-p", required=True,
+                        help="password of vcenter")
+    PARSER.add_argument("--vcUUID", "-d", required=True, default=None,
+                        help="UUID of the relevant vCenter")
+    PARSER.add_argument("--host_id", "-hid", type=int, required=True,
+                        default=None, help="hid of the host")
+    PARSER.add_argument("--reboot_options", "-ro", required=False, default='SAFEREBOOT',
+                        help="Reboot host. Supported values are: SAFEREBOOT, FORCEREBOOT, NEXTREBOOT. "
+                             "Default=SAFEREBOOT")
+    PARSER.add_argument("--exit_maintenance_mode", "-exmm", required=False, default='True',
+                        help="Whether you want exits the maintenance mode after firmware update completes(boolean). "
+                             "Default = True")
+    PARSER.add_argument("--drs_check", "-drs", required=False,
+                        default='True', help="Check DRS is enables for the cluster(boolean). Default=True")
+    PARSER.add_argument("--evacuate_v_ms", "-evm", required=False,
+                        default='True', help="Evacuate powered off and suspended VMs(boolean). Default=True")
+    PARSER.add_argument("--reset_i_drac", "-rid", required=False,
+                        default='False', help="Reset iDRAC and the perform update(boolean). Default=False")
+    PARSER.add_argument("--delete_jobs_queue", "-delj", required=False,
+                        default='True', help="Delete job queue and then perform update(boolean). Default=True")
+    PARSER.add_argument("--enter_maintenance_modetimeout", "-enmm", required=False,
+                        default=60, help="Enter maintenance mode timeout value(int). Default=60mins")
+    PARSER.add_argument("--job_name", "-jn", required=False,
```

Comment:
Job Name not required ? if no user input provided, what will be the job name used?